robot_skills/src/robot_skills/hero.py

```diff
@@ -50,6 +50,63 @@ def __init__(self, wait_services=False):
 
         self.configure()
 
+    def move_to_inspect_pose(self, inspect_target):
+        """
+        This poses the robot for an inspect.
+        :param inspection_position: kdl.Frame with the pose of the entity to be inspected.
+        """
+        # calculate the arm_lift_link which must be sent
+        z_over = 0.3  # height the robot should look over the surface
+        z_hh = 0.9  # height of the robots head at z_arm=0
+        torso_to_arm_ratio = 2  # motion of arm/motion of torso
+        z_head = inspect_target.z() + z_over
+
+        # check whether moving the arm is necessary
+        if z_head < 1.2:
+            rospy.loginfo('Entity is low enough. we dont need to move the arm')
+            return True
+
+        # saturate the arm lift goal
+        z_arm = (z_head - z_hh) * torso_to_arm_ratio
+        if z_arm > 0.69:
+            z_arm = 0.69
+            rospy.logwarn('Warning: looking at excessively high surface')
+        elif z_arm < 0.0:
+            z_arm = 0.0
+            rospy.logwarn('Surface is low enough, we dont need to move the arm.')
+
+        arm = self.get_arm()
+
+        # turn the robot
+        rotation = 1.57
+        rotation_speed = 1
+        rotation_duration = rotation / rotation_speed
+        if arm.has_joint_goal('arm_out_of_way'):
+            pose = arm._arm.default_configurations['arm_out_of_way']
+            pose[0] = z_arm
+            arm._arm._send_joint_trajectory([pose])
+        else:
+            rospy.logwarn('Warning: robot does not have an "arm_out_of_way" joint goal')
+            return False
+        self.base.force_drive(0, 0, rotation_speed, rotation_duration)
+        arm.wait_for_motion_done()
+        return True
+
+    def move_to_hmi_pose(self):
+        """
+        This poses the robot for conversations.
+        """
+        arm = self.get_arm()
+
+        rotation = 1.57
+        rotation_speed = 1
+        rotation_duration = rotation / rotation_speed
+        if arm.has_joint_goal('arm_out_of_way'):
+            arm.send_joint_goal('arm_out_of_way', 0.0)
+            self.base.force_drive(0, 0, rotation_speed, rotation_duration)
+        arm.wait_for_motion_done()
+        return "succeeded"
```

Comment:
Why return a 'succeeded' string? I'd use a bool *if* you check for the return values of base and arm motions and otherwise simply don't return anything.

Make sure to also add this to your docstring.